src/main/java/evaluation/ProgressiveLearner.java

```diff
@@ -196,7 +196,7 @@ private String injectAgentAttributes(String raw) {
     private void runGamesWithAgents() {
         // Run!
         RoundRobinTournament tournament = new RandomRRTournament(agents, gameToPlay, nPlayers,  true, matchups,
-                System.currentTimeMillis(), params);
+                finalMatchups, System.currentTimeMillis(), params);
```

Comment:
this should be 'matchups', not 'finalMatchups'

Reply:
Fixed!